.github/workflows/test.yml

```
@@ -215,3 +215,18 @@ jobs:
       saves_to_disk: true
       disk_suffix: tip
       height_grep_text: 'finished initial sync to chain tip, using gossiped blocks sync_percent=100.* current_height=Height'
+
+  # Test sending transactions with lightwalletd using a Zebra tip state
+  lightwalletd-transactions-test:
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    needs: build
+    uses: ./.github/workflows/gcp-test-deploy.yml
+    with:
+      app_name: lightwalletd
+      test_id: lwd-send-transactions
+      test_description: Test lightwalletd transactions with a Zebra tip state
+      test_variables: '-e TEST_LWD_TRANSACTIONS=1 -e ZEBRA_TEST_LIGHTWALLETD=1 -e ZEBRA_FORCE_USE_COLOR=1 -e ZEBRA_CACHED_STATE_DIR=/var/cache/zebrad-cache'
```

This test needs a lightwalletd directory path, but that directory does not need to exist:

```suggestion
      test_variables: '-e TEST_LWD_TRANSACTIONS=1 -e ZEBRA_TEST_LIGHTWALLETD=1 -e ZEBRA_FORCE_USE_COLOR=1 -e ZEBRA_CACHED_STATE_DIR=/var/cache/zebrad-cache -e LIGHTWALLETD_DATA_DIR=/var/cache/lightwalletd-cache'
```